jwst/datamodels/fits_support.py

```diff
@@ -293,7 +293,7 @@ def _fits_item_recurse(validator, items, instance, schema):
 def _fits_type(validator, items, instance, schema):
     if instance in ('N/A', '#TODO', '', None):
         return
-    return validators._validators.type_draft4(validator, items, instance, schema)
```

Comment:
This method disappears in jsonschema 3.0, but luckily there is a way to access it through a public API that does not change between the versions.